doc/Type/Mu.pod6

```diff
@@ -622,27 +623,24 @@ then do this thing".  For instance,
         say "Verbose option detected in arguments";
     } # OUTPUT: «Verbose option detected in arguments␤»
 
-The C<$verbose-selected> variable in this case contains a
```

Comment:
I don't see why eliminating an explanation helps. It was added last week when an user requested some clarification. If it's  incorrect or confusing, correct it, but eliminating text that might helpful to someone is not really the way to go here.

Reply:
The explanation that was there wasn't correct except to say that the value is "truthy", which is obvious by the example. So IMO it's completely understandable without an explanation and is less content to read.

Reply:
If it's not correct, then the best thing is to change it. That explanation was written to address a particular issue, if it's deleted that issue will be left unaddressed. What's understandable for you might not be for someone else. Unless it's incorrect it's better to leave it there, and if it's incorrect it's better to fix or clarify instead of deleting it.